opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/Ctags.java

```diff
@@ -399,7 +399,7 @@ public Definitions doCtags(String file) throws IOException,
                 int exitValue = ctags.exitValue();
                 // If it is possible to retrieve exit value without exception
                 // this means the ctags process is dead.
-                LOGGER.log(Level.WARNING, "Ctags process exited with exit value {0}",
+                LOGGER.log(Level.FINER, "Ctags process exited with exit value {0}",
```

Comment:
This might actually be a good transformation.

Reply:
Why? The comment says the ctags process is dead.

Reply:
Oh yeah, you're right. There's an exception thrown below. It was folded by GitHub.